plugins/processors/topk/README.md

```diff
@@ -0,0 +1,97 @@
+# TopK Processor Plugin
+
+The TopK processor plugin is a filter designed to get the top series over a period of time. It can be tweaked to do its top k computation over a period of time, so spikes can be smoothed out.
+
+This plugin groups the metrics based on their name and tags, then generates aggregates values across each group base on fields selected by the user. It then sorts these groups based on these aggregations and returns any metric that belongs to a group in the top k (sorted by any of the aggregations). This means that when calculating the top k, more than k metrics may be returned.
+
+If only the very top k metrics are needed, regardless of grouping, the simple_topk setting will force each metric into its own individual group
+
+### Configuration:
+
+```toml
+[[processors.topk]]
+  ## How many seconds between aggregations
+  # period = 10
+
+  ## How many top metrics to return
+  # k = 10
+
+  ## Metrics are grouped based on their tags and name. The plugin aggregates
+  ## the selected fields of these groups of metrics and sorts the groups based
+  ## these aggregations
+
+  ## Over which tags should the aggregation be done. Globs can be specified, in
+  ## which case any tag matching the glob will aggregated over. If set to an
+  ## empty list is no aggregation over tags is done
+  # group_by = ['*']
+
+  ## Wheter or not to also group by metric name
+  # group_by_metric_name = true
```

Comment:
I'm trying to think of a case were this would hurt us, because I feel that it will. But I can't think on anything (though this will force the user to group by some tag, otherwise the plugin will never drop anything)

If I think about an edge case I'll let you know. In the mean time I'll remove the flag